src/resources/views/crud/columns/select.blade.php

```diff
@@ -18,6 +19,7 @@
 
 <span>
     @if(count($column['value']))
+        {!! $column['label'] !!}: 
```

Comment:
Wait. Why? No... why is this needed @munjaldevelopment ? From what I understand what this does is that it outputs the label of the column inside the value, which is useless and ugly:

![CleanShot 2022-11-01 at 11 27 48@2x](https://user-images.githubusercontent.com/1032474/199203024-5b77896a-5d52-48fc-ac7f-799d9c18cd82.png)

Was this done because it's used inside `checklist_dependency` or something? Either way, this is definitely not OK. 

(1) We cannot introduce a breaking change in the `select` column. We cannot introduce a breaking change in ANY existing columns.

(2) Even as a feature... I don't think this is ever needed... showing the label before the value... If this was done for `checklist_dependency`, a better way to do it would have been to pass a `prefix` when loading the select column, rather than modifying the `select` column.